src/main/java/com/booksaw/betterTeams/database/api/Database.java

```diff
@@ -161,6 +161,8 @@ public void executeStatement(String statement, String... placeholders) {
 		try (PreparedStatement ps = connection.prepareStatement(statement)) {
 //			System.out.println("executing: " + ps.toString());
 			ps.executeUpdate();
+			//Shouldn't have negative impact, just need to figure out why it is marked as redundant.
```

Comment:
This is redundant due to how a try-with-resource block works in Java. If you declare the resource as a parameter for the try block, Java will close the resource regardless of if the try block passes or fails.

At the end of the below block the resource is closed no matter if the error is thrown or if the try block executes correctly
```java
try (Resource resource = new Resource()) {
  // do stuff
} catch (Exception e) {
  // do other stuff
}
```

If you do the alternative of creating the resource within the block itself, it will not be closed if the processing throws an error leaving a minor data leak.
```java
try {
  Resource resource = new Resource();
  // do stuff that happens to throw error
  resource.close();
} catch (Exception e) {

}
```